pom.xml

```diff
@@ -115,6 +120,40 @@
         </dependency>
     </dependencies>
     <build>
+    <pluginManagement>
```

Comment:
I'm not sure editor-specific pom file changes are really a good idea. Also, it looks like Eclipse rearranged this file which makes for a very messy diff and makes it far too difficult to determine substantive changes to the file.

Reply:
Thanks for the comment. I typically use SonarQube to analyse every PR and some definitions are required for this phase. I will wait for the project owner feedback.